Makefile

```diff
@@ -71,28 +63,21 @@ Regulome:
 	gzip -dc ${DEST_DIR}/raw/regulome2.csv.gz > ${DEST_DIR}/regulome2.csv
 	gzip -dc ${DEST_DIR}/raw/regulome3.csv.gz > ${DEST_DIR}/regulome3.csv
 	cat ${DEST_DIR}/regulome1.csv ${DEST_DIR}/regulome2.csv ${DEST_DIR}/regulome3.csv > ${DEST_DIR}/regulome.csv
+	rm ${DEST_DIR}/regulome1.csv ${DEST_DIR}/regulome2.csv ${DEST_DIR}/regulome3.csv
 	awk 'BEGIN {FS="\t"} { print $$1,$$2,$$2 + 1,$$4 }' ${DEST_DIR}/regulome.csv | sed -e 's/^chr//' > ${DEST_DIR}/Regulome.bed
 	python scripts/preprocessing/regulome_tidy.py ${DEST_DIR}
 
-#Phenotypes:
-
 d_1000Genomes:
 	mkdir -p ./databases/raw/1000Genomes
 	for url in `cat ./scripts/preprocessing/links.txt`; do wget -nc $${url} -P ./databases/raw/1000Genomes/; done
-process_1000Genomes: 
-	for seq in `cat ./scripts/preprocessing/links.txt | cut -d'/' -f8`; \
-		do gzip -dc databases/raw/1000Genomes/$${seq} | \
-		bgzip -c > databases/1000Genomes/$${seq}.bgz; \
-	done
-
-index_1000Genomes:
-	for seq in `cat ./scripts/preprocessing/links.txt | cut -d'/' -f8`; \
-		do tabix -f databases/1000Genomes/$${seq}.bgz; \
-       	done
-extract_1000Genomes:
-	awk -F "\t" '{if ($$4 == "CEU") print $$1}' ./scripts/preprocessing/igsr_samples.tsv > ./scripts/preprocessing/CEPH_samples.txt
-	mkdir -p ./databases/1000Genomes/CEPH
-	for i in `seq 1 22; echo X; echo Y`; \
-		do vcfkeepsamples ./databases/1000Genomes/ALL.chr$${i}.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz.bgz $$(cat ./scripts/preprocessing/CEPH_samples.txt) > ./databases/1000Genomes/CEPH/CEPH.chr$${i}.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.bgz; \
-	done
+1000Genomes: 
+	for i in `seq 22`; do \
```

Comment:
Ah yes, we definitely need those? I removed all this because I realised I was expensively processing and indexing the 1000 Genomes data containing all the samples, which I later realised was completely unnecessary...